### ISSUE TITLE
Tag and publish releases

### ISSUE THREAD

**Issue by atifsyedali** [WebAssembly/wasi-keyvalue#50]
Please create releases and tag the repository with each version. The recent change from `wasi:keyvalue@0.2.0-draft` to `wasi:keyvalue@0.2.0-draft2` means that all `wit-deps update` effectively pulls in the `https://github.com/WebAssembly/wasi-keyvalue/archive/main.tar.gz` file with `0.2.0-draft2`, but because some open-source implementations are still on `0.2.0-draft`, this mismatch breaks existing solutions that are pulling in both the latest keyvalue WIT and the open source implementation on older version of the WIT.

**Comment by Mossaka** [comment]
yeah we need to do a better job at tagging each version. I will do that

**Comment by Mossaka** [comment]
Just tagged the `v0.2.0-draft` https://github.com/WebAssembly/wasi-keyvalue/tree/v0.2.0-draft

**Comment by pchickey** [comment]
Can you post `v0.2.0-draft2` as well with the current contents of main? I'm working on an implementation and would like to manage the wit with `wkg`. Right now the ghcr.io registry only has -draft.

**Comment by Mossaka** [comment]
We are holding on publishing `v0.2.0-draft2` because I believe @ricochet and @thomastaylor312 are going to push a change to address https://github.com/WebAssembly/wasi-keyvalue/issues/47

**Comment by thomastaylor312** [comment]
Yeah we fell behind on this (obviously). I also have some remaining work to come bring in an automatic publishing pipeline here, but maybe it would just be easier especially now that we're working on https://github.com/bytecodealliance/wkg-github-action

**Comment by mikkelhegn** [comment]
Is there a `0.1.0` version of this spec, which was ever released? It's unclear based on history and tags/releases?

**Comment by pchickey** [comment]
@mikkelhegn there is no version 0.1.0 of this spec. By convention, the first release version of this spec would be 0.2.x, to match with the rest of the WASI 0.2 specs, which are on a [release train](https://bytecodealliance.org/articles/WASI-0.2.1). Before release, which requires a WASI subgroup vote, proposal champions can tag versions 0.2.0-draftN whenever they feel its appropriate.

**Comment by mikkelhegn** [comment]
Thanks for the explanation @pchickey.
